example/src/test/scala/monocle/function/AtExample.scala

```diff
@@ -34,4 +38,21 @@ class AtExample extends MonocleSuite {
     (Set(1, 2, 3) applyLens at(2) set None)     shouldEqual Set(1, 3)
   }
 
+  test("at creates a Lens from Int to one of its bit") {
+    (3 applyLens at(0: IntBits) get) shouldEqual true   // true  means bit is 1
```

Comment:
Just playing with that. It seems the ascription is necessary because it guides the implicit search for the correct `At` instance.